localstack/aws/connect.py

```diff
@@ -49,7 +49,6 @@ def my_patch(fn, self, **kwargs):
         patched_kwargs = {
             **kwargs,
             "WaiterConfig": {
-                # TODO: make these configurable
```

Comment:
This change doesn't seem to be related, but if it's on purpose, it's okay to sneak this one in ;)